lib/sqlalchemy/sql/selectable.py

```diff
@@ -4698,8 +4729,8 @@ def _get_display_froms(
                 f
                 for f in froms
                 if f
```

Comment:
**mike bayer** (zzzeek) wrote:

remove both

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/sqlalchemy/+/4325